diff -Naur gparted-0.17.0.orig/data/com.ubuntu.pkexec.gparted.policy gparted-0.17.0/data/com.ubuntu.pkexec.gparted.policy
--- gparted-0.17.0.orig/data/com.ubuntu.pkexec.gparted.policy	2014-02-06 04:08:40.000000000 +0000
+++ gparted-0.17.0/data/com.ubuntu.pkexec.gparted.policy	2014-02-06 04:14:58.283728498 +0000
@@ -10,7 +10,7 @@
     <defaults>
       <allow_any>auth_admin</allow_any>
       <allow_inactive>auth_admin</allow_inactive>
-      <allow_active>auth_admin</allow_active>
+      <allow_active>yes</allow_active>
     </defaults>
     <annotate key="org.freedesktop.policykit.exec.path">/usr/sbin/gparted</annotate>
     <annotate key="org.freedesktop.policykit.exec.allow_gui">true</annotate>
diff -Naur gparted-0.17.0.orig/data/com.ubuntu.pkexec.gparted.policy.in gparted-0.17.0/data/com.ubuntu.pkexec.gparted.policy.in
--- gparted-0.17.0.orig/data/com.ubuntu.pkexec.gparted.policy.in	2014-02-06 04:08:40.000000000 +0000
+++ gparted-0.17.0/data/com.ubuntu.pkexec.gparted.policy.in	2014-02-06 04:15:06.664014435 +0000
@@ -10,7 +10,7 @@
     <defaults>
       <allow_any>auth_admin</allow_any>
       <allow_inactive>auth_admin</allow_inactive>
-      <allow_active>auth_admin</allow_active>
+      <allow_active>yes</allow_active>
     </defaults>
     <annotate key="org.freedesktop.policykit.exec.path">/usr/sbin/gparted</annotate>
     <annotate key="org.freedesktop.policykit.exec.allow_gui">true</annotate>
